### DISCUSSION TITLE
每日美图分享（1）

### DISCUSSION THREAD

**Discussion by qingshuiglaze** [share121/inter-knot#5215] [崩坏：星穹铁道]
![108902728_p0](https://github.com/user-attachments/assets/774dc18f-6562-498b-ada1-053527ef6a9f)
![112473057_p0](https://github.com/user-attachments/assets/1b730292-573f-419d-bc20-93932dd379d1)
![89306326_p0](https://github.com/user-attachments/assets/2236051d-0246-492f-8d6d-864755545776)
![115908961_p0](https://github.com/user-attachments/assets/5ea7ff73-c490-4a69-b70d-e1bd8fa54503)
![113296953_p0](https://github.com/user-attachments/assets/e7b52e4c-fea4-4981-b370-8e7274b94e3a)

**Comment by aino2333** [comment]
豪堪！

**Comment by anlaneee** [comment]
qq
